scripts/ntrip_ros.py

```diff
@@ -167,13 +179,23 @@ def stop(self):
     self.destroy_node()
 
   def subscribe_nmea(self, nmea):
-    # Just extract the NMEA from the message, and send it right to the server
-    self._client.send_nmea(nmea.sentence)
+    # Cache the latest NMEA sentence
+    self._latest_nmea = nmea.sentence
+
+  def send_rtcm_and_nmea(self):
+    # Send cached NMEA data if available
+    if self._latest_nmea is not None:
+      self._client.send_nmea(self._latest_nmea)
 
-  def publish_rtcm(self):
+    # Request and publish RTCM data
     for raw_rtcm in self._client.recv_rtcm():
       self._rtcm_pub.publish(self._create_rtcm_message(raw_rtcm))
 
+    # Publish a confirmation message to indicate the send_rtcm_and_nmea call
+    confirmation_msg = String()
+    confirmation_msg.data = "RTCM request and NMEA receive set at rate: {} Hz".format(1.0 / self.rtcm_request_rate)
+    self._rate_confirm_pub.publish(confirmation_msg)
```

Comment:
Same as above. I would be okay having this as a debug log, but would rather not have an extra string publisher